### ISSUE TITLE
Does gherkin-testcafe support Scenario Outline?

### ISSUE THREAD

**Issue by ZeniStiks** [kiwigrid/gherkin-testcafe#37]
Hi,
I'm trying to write a simple test and I need to use Scanario Outline
```
Feature: feature 1

  Scenario Outline: scenario
    Given the user has <role> in app
    When action
    Then result

    Examples:
      | role  |
      | Admin |
      | User |
```
`
Given(/^the user has (.*) in app$/, async (t, role) => {
console.log(role);
});
`
After running this output I get is `<role>` and only one scenario is run when it should be two
How do I access my example values?
Tried examples in [Support for data tables #14](https://github.com/kiwigrid/gherkin-testcafe/issues/14) but it's not working :/

**Comment by Lukas-Kullmann** [comment]
Hey @ZeniStiks,

this should be working. We use Scenario Outlines ourselves.
It sould log twice. First, `['Admin'] ` and `['User']` second. (Note that your argument `role`is an array as of `gherkin-testcafe@2.0.0`)

Unfortunately, I can only investigate on Monday. So have a nice weekend until then.

**Comment by ZeniStiks** [comment]
Yes, I understand that it's an array. Also a correction, output is `[ '<role>' ]`

**Comment by Lukas-Kullmann** [comment]
Well, that seems like a bug. I'm sorry for the inconvenience. But as I said, I can only investigate on Monday.

**Comment by ZeniStiks** [comment]
No problem, have a nice weekend!

**Comment by ZeniStiks** [comment]
A small update. I was messing around with different versions of gherkin-testcafe and it looks like scenario outlines are not working in latest version only(2.3.0)
I hope this will provide some directions for the roots on this issue

**Comment by Lukas-Kullmann** [comment]
Hey @ZeniStiks,

I investigated the issue and found that it actually was a bug. I released v2.3.1 with a fix.
